app.js

```diff
@@ -0,0 +1,221 @@
+  .addFields(
+    {
+      name: "Categories",
+      value:
+        "Wildcard, Living Better, Business & Tech, History & Culture, Science & Nature",
+    },
+    { name: "\u200B", value: "\u200B" },
+    {
+      name: "Commands",
+      value: command,
+      inline: true,
+    }
+  )
+  .setTimestamp()
+  .setFooter("Happy Reading", "https://i.imgur.com/vugPtoT.png");
+
+const rule = new schedule.RecurrenceRule();
+rule.dayOfWeek = [0, new schedule.Range(0, 6)];
+rule.hour = 10;
+rule.minute = 00;
+var startDay = 0;
+var endDay = 6;
+
+//creating a empty cron expression
+var cronExpression = `${rule.minute} ${rule.hour} * * ${startDay}-${endDay}`;
+
+var dailyUpdatesChannel = null;
+
+// fetching random article
+function fetchRandomArticle(category) {
+  lengthOfArticle = data[category].length;
+  randomArticlePosition = Math.floor(Math.random() * lengthOfArticle);
+  article = data[category][randomArticlePosition];
+  if (category == "WILDCARD") {
+    try {
+      articleLink = article["Link to Article (from Article List)"][0];
+    } catch (error) {
+      console.log(error);
+      dailyUpdatesChannel.send("An error occured, could you please try again");
+    }
+  } else {
+    try {
+      articleLink = article["Link to Article"];
+    } catch (error) {
+      console.log(error);
+      dailyUpdatesChannel.send("An error occured, could you please try again");
+    }
+  }
+  data[category] =
+    data[category][(0, randomArticlePosition)] +
+    data[category][(randomArticlePosition + 1, lengthOfArticle)];
+  data[category] =
+    _.slice(data[category], 0, randomArticlePosition) +
+    _.slice(data[category], randomArticlePosition + 1, lengthOfArticle);
+  return articleLink;
+}
+
+// resetting schedule after changing timings
+function resetScheduler() {
+  const job = schedule.scheduleJob(rule, function () {
+    dailyUpdatesChannel.send(fetchRandomArticle("WILDCARD"));
+    cronExpression = `${rule.minute} ${rule.hour} * * ${startDay}-${endDay}`;
+  });
+}
+
+//Playing Message
+client.on("ready", async () => {
+  console.log(
+    `${client.user.username} is online on ${client.guilds.cache.size} servers!`
+  );
+
+  client.user.setActivity("Article", { type: "PLAYING" });
+
+  resetScheduler();
+});
+
+// handling on message events
+client.on("message", (msg) => {
+  dailyUpdatesChannel = msg.guild.channels.cache.find(
+    (channel) => channel.name === "readsomethinggreat"
+  );
+  cronExpression = `${rule.minute} ${rule.hour} * * ${startDay}-${endDay}`;
+
+  if (msg.author.bot) return;
+
+  if (msg.content === "get help") {
+    msg.channel.send({ embeds: [exampleEmbed] });
+  }
+
+  if (
+    msg.content.startsWith("get article") ||
+    msg.content.startsWith("Get article")
+  ) {
+    msgRecievied = msg.content.split(" ");
+    if (msgRecievied.length == 2) {
+      msg.channel.send(
+        "```Here are the categories to read upon:" +
+          "\n" +
+          "1. Wildcard" +
```

Comment:
you have defined categories in line 20 before, make this message from them dynamically?

Reply:
Actually, they are not the same, these are somewhat formatted but there are just for accessing data without any name errors after capitalizing.